ignores/nl_NL/tickets.txt

```diff
@@ -0,0 +1,7 @@
+msgid "SUDOMEMO_CITIZENSHIP"
+msgid "TICKET_NAME_green_panda_ticket"
+msgid "TICKET_NAME_red_bird_ticket"
+msgid "TICKET_NAME_blue_dog_ticket"
+msgid "TICKET_NAME_purple_bunny_ticket"
+msgid "TICKET_NAME_golden_neko_ticket"
+msgid "TICKET_NAME_golden_neko_ticket"
```

Comment:
Ignore files should just have the msgids in them, for example the line:
`msgid "SUDOMEMO_CITIZENSHIP"`
should be:
`SUDOMEMO_CITIZENSHIP`